templates/deployment.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "kafka-manager.fullname" . }}
+  labels:
+    app: {{ template "kafka-manager.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ default 1 .Values.replicaCount }}
```

Comment:
You can remove `default` as it is defined in `values.yaml`

Reply:
Done in 86f4166